core/src/main/java/com/exadel/etoolbox/backpack/core/servlets/DeletePackageServlet.java

```diff
@@ -0,0 +1,56 @@
+package com.exadel.etoolbox.backpack.core.servlets;
+
+import com.exadel.etoolbox.backpack.core.services.pckg.DeletePackageService;
+import org.apache.sling.api.SlingHttpServletRequest;
+import org.apache.sling.api.SlingHttpServletResponse;
+import org.apache.sling.api.request.RequestParameter;
+import org.apache.sling.api.servlets.SlingAllMethodsServlet;
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Reference;
+
+import javax.servlet.Servlet;
+import javax.servlet.http.HttpServletResponse;
+
+/**
+ * Serves as the network endpoint for user requests that delete package<br><br>
```

Comment:
(cosmetic) Serves as the network endpoint for user requests for deleting a package (or: to delete a package)
Also, do you need both `<br><br>` and `<p>` afterwards? Looks like to many spaces of different kinds all at once)